tools/adjust-db/README.md

```diff
@@ -0,0 +1,23 @@
+# adjust-db tool
+This is a tool that should only be used for testing purposes.  
+It is intended as a collection of commands that perform small db modifications.
+
+
+## change-db-kind
+Changes DbKind of a DB described in config (cold or hot).  
```

Comment:
config -> command line arguments

Reply:
DB for which I am changing kind is described in the config. Or do you mean --home argument? If so, fair. But I don't know how to work that in without creating confusion.